src/buffer/out/TextAttribute.cpp

```diff
@@ -93,6 +119,7 @@ void TextAttribute::SetFromLegacy(const WORD wLegacy) noexcept
     BYTE bgIndex = static_cast<BYTE>(wLegacy & BG_ATTRS) >> 4;
     _foreground = TextColor(fgIndex);
     _background = TextColor(bgIndex);
+    //SetMetaAttributes(wLegacy); // [danthom] should this be here?

```

Comment:
>//SetMetaAttributes(wLegacy); // [danthom] should this be here? [](start = 4, length = 63)

Opinions?

Reply:
I have a sneaking suspicion that it _used to_ be that way actually, but that broke some other scenario. 

What scenario does adding that line fix?

Does adding that line cause any unittests to fail?

Reply:
No; it doesn't seem to make a difference either way. I just happened to notice when comparing `SetLegacyAttributes` and `SetFromLegacy`.

If your suspicion turns to full realization, let me know, and I can add a comment there explaining that `SetMetaAttribute` is *deliberately* skipped here.

(else I'll just remove this line completely)